apps/web/composables/useConfigurationDrawer/useConfigurationDrawer.ts

```diff
@@ -1,25 +1,30 @@
-import { getTailwindColorsOklch, oklchToRgb } from '~/utils/tailwindHelper';
+import { getPaletteFromColor } from '~/utils/tailwindHelper'; // Adjust the import based on your file structure
```

Comment:
```suggestion
import { getPaletteFromColor } from '~/utils/tailwindHelper';
```
**todo**: Remove comment. I think having to change the imports based on the file location is obvious to any developer.